tutorials/20_particles_tutorial.md

```diff
@@ -0,0 +1,94 @@
+\page particles Particles
+
+This example shows how to include a particle emitter in the scene.
+
+## Compile and run the example
+
+Create a build directory and use `cmake` and `make` to compile the code:
+
+```{.sh}
+cd ign-rendering/examples/particles_demo
+mkdir build
+cd build
+cmake ..
+make
+```
+Execute the example:
+
+```{.sh}
+./particles_demo
+[Msg] Loading plugin [ignition-rendering4-ogre]
+Engine 'optix' is not supported
```

Comment:
how about we remove `optix` from this demo so this msg does not get printed out? so mainly [this line](https://github.com/ignitionrobotics/ign-rendering/blob/master/examples/particles_demo/Main.cc#L151)

Reply:
Agreed! Done in 3a233f6.